### PR TITLE
Route controllers

### PR DESCRIPTION
**Pull request by seanmcneilly2**

### CODE REVIEW

File: app/server/controllers/accountSummaryController.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import shouldSetInitialScenario from '../middleware/shouldSetInitialScenario';
+import setSessionScenario from '../middleware/setSessionScenario';
+import HistoryService from '../services/history';
+
+exports.getAccountSummary = function getAccountSummary(scenarioService, permissionsService, req, res) {
```
Comment (ammarhaiderbjss) [inline app/server/controllers/accountSummaryController.js]: can we use `export function getAccountSummary` ?

File: app/server/helpers/accountHistorySearch.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+export default function (sessionData) {
+	let selectedOption = '';
+	let selectedStartDate = '';
+	let selectedEndDate = '';
+	let startDate = '';
+	let endDate = '';
+
+	if (sessionData !== undefined) {
+		selectedOption = sessionData['select-box'];
+		selectedStartDate = sessionData['date-picker-0'];
```
Comment (danielchildsbjss) [inline app/server/helpers/accountHistorySearch.js]: Can we use date-picker-start and date-picker-end, we are using 0,1 and 2 as ids and it's not always clear.

File: app/server/helpers/accountHistorySearch.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+		selectedEndDate = sessionData['date-picker-1'];
+
+		startDate = DateSplitter(selectedStartDate);
+		endDate = DateSplitter(selectedEndDate);
+
+		if (startDate === undefined) {
```
Comment (ammarhaiderbjss) [inline app/server/helpers/accountHistorySearch.js]: Another JS quirk - `undefined` can be reassigned :(
For these checks can we use `typeof foo === 'undefined'`?

File: app/server/helpers/accountHistorySearch.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import DateSplitter from './dateSplitter';
+
+export default function (sessionData) {
+	let selectedOption = '';
+	let selectedStartDate = '';
+	let selectedEndDate = '';
+	let startDate = '';
+	let endDate = '';
+
+	if (sessionData !== undefined) {
```
Comment (ammarhaiderbjss) [inline app/server/helpers/accountHistorySearch.js]: Do we need to handle the case when sessionData isn't provided?

**Pull request merged**